charts/redpanda/ci/09-initcontainers-resources-values.yaml

```diff
@@ -73,13 +73,13 @@ statefulset:
         - name: test-extra-volume
           mountPath: /fake/lifecycle
     extraInitContainers: |-
-      - name: "test-init-container"
-        image: "mintel/docker-alpine-bash-curl-jq:latest"
-        command: [ "/bin/bash", "-c" ]
-        args:
-          - |
-            set -xe
-            echo "Hello World!"
+    - name: "test-init-container"
```

Comment:
What lead to this being reformatted? 🤔

Reply:
that's a glitch; I was checking a bug report. I'll drop this.